cryptoki/src/types/mechanism/mod.rs

```diff
@@ -117,6 +124,8 @@ pub enum Mechanism {
     // ECC
     /// EC key pair generation
     EccKeyPairGen,
+    /// ECDH
+    Ecdh(ecdh::EcdhDeriveParams),
```

Comment:
Maybe the name `Ecdh1Derive` (as the wrapper for `CKM_ECDH1_DERIVE`) would be better here?